### DISCUSSION TITLE
Does this support import and export annotations?

### DISCUSSION THREAD

**Discussion by jhunexjun** [wojtekmaj/react-pdf#1648] [Q&A]
1. Does it support importing and exporting annotations? I'm not seeing sample.
2. How can its instance be called. like reactpdfObj.getAnnotation().
3. Using the instance, can we it generate its blob file with option to get the flat file or not. I mean to get the flat file that will be final after annotation is applied. If not flat file, the annotation is removed.

**Comment by wojtekmaj** [comment]
React-PDF is built around Mozilla's PDF.js, so you can use `page.getAnnotations()` where you receive `page` from `Page` component's `onLoadSuccess` callback.
